pkg/meta/store_test.go

```diff
@@ -117,20 +208,41 @@ func (d *Demo3) Key() string {
 	return d.B
 }
 
-var boltdbStore *Store
+func initBoltDBStore(dbFile string) (*Store, error) {
+	boltdbCfg := Config{
+		Driver:  "boltdb",
+		BaseDir: dbFile,
+		Buckets: []Bucket{
+			{"boltdb", reflect.TypeOf(Demo3{})},
+		},
+	}
 
-var boltdbCfg = Config{
-	Driver:  "boltdb",
-	BaseDir: "/tmp/bolt.db",
-	Buckets: []Bucket{
-		{"boltdb", reflect.TypeOf(Demo3{})},
-	},
+	return NewStore(boltdbCfg)
+}
+
+func ensureFileNotExist(file string) error {
+	_, err := os.Stat(file)
+	if err == nil {
+		os.Remove(file)
```

Comment:
Actually, `file` is a directory path, using `os.RemoveAll` instead.

Reply:
I am afraid, `file` is a file path